config/image_tools_parameters.yaml

```diff
@@ -0,0 +1,22 @@
+# The usual format of the yaml file would be something like this:
+#cam2image:
```

Comment:
nit: add an space betwen the `#` and the initiation of the comment. Here and there.

Reply:
Done.